typespec-tests/tsp/protocol-api-convenient-api.tsp

```diff
@@ -97,6 +97,19 @@ model ResourceJ {
   type: string;
 }
 
+@trait
+model ListTrait {
+  @traitContext(TraitContext.List)
+  reqeust: {
```

Comment:
typo?

Reply:
yeah, typo. The name seems not have effect at the first place.